package.json

```
@@ -175,6 +175,7 @@
 		"test:php": "node ./tools/local-env/scripts/docker.js run -T php composer update -W && node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit",
 		"test:e2e": "node ./tests/e2e/run-tests.js",
 		"test:visual": "node ./tests/visual-regression/run-tests.js",
-		"sync-gutenberg-packages": "grunt sync-gutenberg-packages --dev"
+		"sync-gutenberg-packages": "grunt sync-gutenberg-packages",
+		"postsync-gutenberg-packages": "grunt wp-packages:sync-stable-blocks && grunt build --dev"
```

I just got confused by a PHP error – turns out this does everything we need to commit, but it doesn't prepare the local build for testing. Let's also run the regular build:

```suggestion
		"postsync-gutenberg-packages": "grunt wp-packages:sync-stable-blocks && grunt build --dev && grunt build"
```

---

I never know which commands you need to run 😅 

We had only `grunt build --dev` before.